website/cue/reference/components/sources/opentelemetry.cue

```diff
@@ -203,6 +209,13 @@ components: sources: opentelemetry: {
 				[OpenSSL configuration file](\(urls.openssl_conf)). The file location defaults to
 				`/usr/local/ssl/openssl.cnf` or can be specified with the `OPENSSL_CONF` environment variable.
 				"""
+		},
+		traces: {
+			title: "Ingest OTLP traces"
+			body: """
+				Vector now support ingest opentelemetry trace in proto format(JSON is not supported). The trace
+				data is then converted as log, and you can use vrl to remap it to whatever you want.
```

Comment:
```suggestion
				Vector now support ingest opentelemetry trace in proto format(JSON is not supported). The trace
				data is then converted as log, and you can use vrl to remap it to whatever you want.
```

I don't think this is true, is it? It is a trace event (granted it shares the same internal structure as a log). I would suggest just dropping this since we don't have a "how it works" section for the log data type.

Reply:
```suggestion
				Trace support is experimental and subject to change as Vector has no strongly-typed structure for traces internally. Instead traces are stored as a key/value map similar to logs. This may change in the future to be a structured format.
```

Or I'm remembering now that we wanted to explicitly call this out as experimental. Could we change it to something like this?

Reply:
I think we should add a note up under `notices` here too that says `Ingestion of traces is experimental`.